docs/apache-airflow-providers-amazon/operators/lambda.rst

```diff
@@ -68,10 +100,10 @@ Sensors
 
 .. _howto/sensor:LambdaFunctionStateSensor:
 
-Wait on an Amazon Lambda function state
-=======================================
+Wait on an AWS Lambda function deployment state
```

Comment:
Thanks for fixing this.   Amazon/AWS Service names are so confusing :+1:

Reply:
"There are only two hard things in Computer Science: cache invalidation and naming things"